src/scss/vendors/_vendors.scss

```diff
@@ -3,23 +3,50 @@
 // UIO
 
 // To adapt to UIO high contrast themes
-[class^="gpii-ext-theme"],
+
+// TODO: Address style lint errors when line below is uncommented
+// [class^="gpii-ext-theme"],
```

Comment:
When including the selector `.gpii-ext-theme-#{$theme}` for UIO + themes the following linting errors occur:

```
✖ stylelint found some errors. Please fix them and try committing again.
 
src/scss/vendors/_vendors.scss
20:3  ✖  Expected selector "[class^="gpii-ext-theme"] footer .footer-logos a" to come before selector "[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) footer .funders a"                                                                                            no-descending-specificity
27:2  ✖  Expected selector "[class^="gpii-ext-theme"] #defaultContainer a:hover" to come before selector "[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) #defaultContainer a:focus"                                                                                 no-descending-specificity
29:2  ✖  Expected selector "[class^="gpii-ext-theme"] #defaultContainer header .site-nav-wrapper .site-nav nav a:hover" to come before selector "[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) #defaultContainer header .site-nav-wrapper .site-nav nav a:focus"   no-descending-specificity
30:2  ✖  Expected selector "[class^="gpii-ext-theme"] #defaultContainer .content-wrapper .tags-info .tags a:focus" to come before selector "[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) #defaultContainer header .site-nav-wrapper .site-nav nav a:focus"        no-descending-specificity
30:2  ✖  Expected selector "[class^="gpii-ext-theme"] #defaultContainer .content-wrapper .tags-info .tags a:focus" to come before selector "[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) #defaultContainer header .site-nav-wrapper .site-nav nav a:hover"        no-descending-specificity
31:2  ✖  Expected selector "[class^="gpii-ext-theme"] #defaultContainer .content-wrapper .tags-info .tags a:hover" to come before selector "[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) #defaultContainer header .site-nav-wrapper .site-nav nav a:focus"        no-descending-specificity
31:2  ✖  Expected selector "[class^="gpii-ext-theme"] #defaultContainer .content-wrapper .tags-info .tags a:hover" to come before selector "[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) #defaultContainer header .site-nav-wrapper .site-nav nav a:hover"        no-descending-specificity
31:2  ✖  Expected selector "[class^="gpii-ext-theme"] #defaultContainer .content-wrapper .tags-info .tags a:hover" to come before selector "[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) #defaultContainer .content-wrapper .tags-info .tags a:focus"             no-descending-specificity
34:2  ✖  Expected selector "[class^="gpii-ext-theme"] .card:hover" to come before selector "[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) .card:focus"                                                                                                             no-descending-specificity
36:2  ✖  Expected selector "[class^="gpii-ext-theme"] button:hover" to come before selector "[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) button:focus" 
```

Reply:
While working on #266, I found CSS attached with the generic selector `[class^="gpii-ext-theme"], [class^="fl-theme"]:not(.fl-theme-prefsEditor-default)` sometimes are not applied when another UIO style that is attached with a specific theme selector kicks in. The solution (#269) is to hook up CSS with each specific theme selector.

From the lesson learnt there, I feel the same rule applies to other CSS you have here too. It means to move all css inside `[class^="gpii-ext-theme"], [class^="fl-theme"]:not(.fl-theme-prefsEditor-default)` to the mixin function. This change will fix this linting error too.

Reply:
@cindyli I have moved only the selectors with a `currentColor` value to `_mixins.scss` file. Let me now if this is a valid a solution for this linting problem and I will request another review

Reply:
I tested with your update. Unfortunately, the similar issue as #266 appears with other css that are not moved from the generic theme into specific theme selector.

Using hiding background image of home page cards as an example:

Turn on "enhance inputs" -> switch to another theme -> the background images of cards show up

These images should not be shown on contrast themes.

<img width="1087" alt="Screen Shot 2020-06-08 at 2 01 37 PM" src="https://user-images.githubusercontent.com/572877/84064681-0a840e00-a991-11ea-9012-f82ac754bd2f.png">